### PR TITLE
Set DOCKER_NETWORK_OPTIONS via /etc/sysconfig/docker-network

### PR DESCRIPTION
**Pull request by sdodson**

### CODE REVIEW

**Comment by sdodson** [comment]
Fixes #52 I think testing now

File: rel-eng/openshift-sdn-node.sysconfig [hunk 1]
```diff
@@ -55,4 +55,4 @@ OPTIONS=
 # The $DOCKER_OPTIONS variable is used to overwrite the $OPTIONS
 # variable in the /etc/sysconfig/docker file. This is a pretty ugly
 # hack, but whatevs
```
Comment (rajatchopra) [inline rel-eng/openshift-sdn-node.sysconfig]: @sdodson Minor one. Update the comment above?

**Comment by rajatchopra** [comment]
Runs fine with manual testing. Will re-test with rpm.
Ready for merge post comment-fix above.

File: rel-eng/openshift-sdn-node.sysconfig (hunk 1 above)
Comment (sdodson) [inline rel-eng/openshift-sdn-node.sysconfig]: Should be fixed now, also linked to more elegant solution for anyone willing to invest more time.

**Pull request merged**